cocotb/scheduler.py

```diff
@@ -458,6 +458,17 @@ def _event_loop(self, trigger):
                     # happened in gh-957)
                     del coro
 
+                # Handle any newly queued coroutines that need to be scheduled
+                while self._pending_coros:
+                    task = self._pending_coros.pop(0)
+                    if _debug:
+                        self.log.debug("Scheduling queued coroutine %s" % (task._coro.__qualname__))
+                    self._schedule(task)
+                    if _debug:
+                        self.log.debug("Scheduled queued coroutine %s" % (task._coro.__qualname__))
+
+                    del task
```

Comment:
Are we sure this is needed?

Reply:
I'm not sure, I just followed the pattern above it from #958.

Reply:
Alright. It can stay.